helper/impls/helper-impl-111/src/main/java/com/linkedin/avroutil1/compatibility/avro111/FieldBuilder111.java

```diff
@@ -115,6 +117,34 @@ public Schema.Field build() {
     return result;
   }
 
+  @Override
+  public FieldBuilder addProp(String propName, String jsonObject) {
+    ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
should probably explicitly check for null arguments

Reply:
Are you okay with using the `@Nonnull` annotation instead?

Reply:
no. for several reasons:

1. there's no single authoritative source of those - plenty of projects like guava define them, but those would be extra dependencies (and guava ...)
2. nothing guarantees they are enforced by user IDE

so i'd go the old fashioned route and check the inputs